src/mod/l2_debug_com.c

```diff
@@ -19,6 +19,8 @@ The escaped data is Xored by 0x20.
 */
 #include "l2_debug_com.h"
 
+#include "modules_globals.h"
+uint16_t LAST_STARTED_MODULE;

```

Comment:
keep it as is at this moment. But I plan to move this 'debug feature' of current running module somewhere to the base / mainloop code... (see #17 )